en/apispec/openapi.json

```diff
@@ -3218,199 +2617,2447 @@
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/Authorization"
+          },
+          {
+            "$ref": "#/components/parameters/Date"
+          },
+          {
+            "$ref": "#/components/parameters/Accept-Encoding"
+          },
+          {
+            "$ref": "#/components/parameters/Accept"
+          },
+          {
+            "$ref": "#/components/parameters/On-Behalf-Of"
+          },
+          {
+            "$ref": "#/components/parameters/Content-Type"
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "customerID": {
+                    "type": "string",
+                    "description": "ID of the customer in ngAdmin DB. The customer must be a child of the caller. Each customer is allowed to have one serviceQuota only. The customer must have “HDT” in its products list before a service quota can be created for the customer."
+                  },
+                  "status": {
+                    "type": "string",
+                    "description": "The status of a customer. Can be “Trial”, “Active” or “Suspended”. Default set to “Suspended”.\n\nThe accounts with “trial” status are subject to certain restrictions, e.g., limited trial period and minimum resource availability.  \n\nIf the status is set to “suspended”, any requests from the customer will be rejected by the HDT API server. Note that this status is related to HDT service only, it doesn't affect other products subscribed by the customer."
+                  },
+                  "trialDeadline": {
+                    "type": "string",
+                    "description": "Date-time in RFC 3339 format indicating when the trial expires, e.g. 2021-07-01T15:00:00Z.\nRequired only when the status is set to “trial”. Ignored otherwise. Cannot be a date-time in the past. The trial cannot last longer than 30 days after the trial starts.\n\nWhen trialDeadline expires, the status will be set to “Suspended” and all existing HDT transports will be inactive, after 30 more days of at “suspended” status, all transports for the customer will be deleted."
+                  },
+                  "billUsingStripe": {
+                    "description": "Default to false. A flag indicating whether this customer will be invoiced using Stripe. When set to true, the billing module which integrates Stripe will take care of invoicing for this customer.\n0 as false, 1 as true.",
+                    "type": "integer"
+                  },
+                  "contractID": {
+                    "type": "string",
+                    "description": "ID of the contract signed for HDT service subscription.\n\nRequired when “billUsingStripe” is set to true. The billing module will use this ID to query billing items and prices for the customer at SAP."
```

Comment:
The second sentence is exposing our internal implementation.
Why does the customer need to know anything about our use of SAP?  I suggest reducing references to internal systems.  We may not use SAP forever.